chapters/annotations.tex

```diff
@@ -2324,6 +2324,15 @@ \subsection{Licensing}\label{licensing}
 \end{example}
 
 
+\section{License Texts}\label{license-texts}
+
+For a top-level class the {\lstinline!annotation(License="modelica:/ModelicaLibraryName/Resources/Licenses/MyLicense.txt")!}\annotationindex{License}, gives the license files for the class.
+The annotation may give a scalar value or an array of values.
+The license files are human-readable files containing the license conditions.
+External functions may also contain this annotation (with similar contents), see \cref{annotations-for-external-libraries-and-include-files}.
```

Comment:
Better with _content_?
```suggestion
External functions may also contain this annotation (with similar content), see \cref{annotations-for-external-libraries-and-include-files}.
```

Reply:
Yes, if we consider the meaning of the annotation - not the letters.